paper-typeahead-input.html

```diff
@@ -449,6 +450,14 @@
         // }
       }
     },
+    _candidatesUpdated:function(candidates){
+      if(this.inputValue != undefined){
```

Comment:
I disagree with adding the typeof check since it just masks the reference error. We can make it check for null if you'd like.
```JS
if(this.inputValue != null){